azext_iot/central/_help.py

```diff
@@ -520,35 +661,67 @@ def _load_central_command_help():
                 --device-id {deviceid}
                 --interface-id {interfaceid}
                 --command-name {commandname}
+
+            - name: Show component command response
+              text: >
+                az iot central device command history
+                --app-id {appid}
+                --device-id {deviceid}
+                --component-name {componentname}
+                --command-name {commandname}
+
+            - name: Show module component command response
+              text: >
+                az iot central device command history
+                --app-id {appid}
+                --device-id {deviceid}
+                --module-name {modulename}
+                --component-name {componentname}
+                --command-name {commandname}
         """
 
     helps[
         "iot central device command run"
     ] = """
             type: command
             short-summary: Run a command on a device and view associated response. Does NOT monitor property updates that the command may perform.
-            long-summary: |
-                Note: payload should be nested under "request".
-                i.e. if your device expects the payload in a shape {"key": "value"}
-                payload should be {"request": {"key": "value"}}.
-                --content can also be pointed at a filepath like this (.../path/to/payload.json)
+            long-summary: --content can be inline json or file path.
             examples:
-            - name: Run command response
+            - name: Run command with inline payload. Payload should be under "request" in json
               text: >
                 az iot central device command run
                 --app-id {appid}
                 --device-id {deviceid}
                 --interface-id {interfaceid}
                 --command-name {commandname}
-                --content {payload}
+                --content '{"request": {payload}}'
 
-            - name: Short Run command response
+            - name: Short Run command with json payload path.
               text: >
                 az iot central device command run
                 -n {appid}
                 -d {deviceid}
                 -i {interfaceid}
                 --cn {commandname}
+                -k {.../path/to/payload.json}
```

Comment:
you can just have -k {payload_path}

does the file also need to have the "request" in it?

Reply:
Okay. payload is either under 'request' in inline json or in json file.

Reply:
for clarification: don't have 'request' in file